system-addon/lib/ActivityStreamMessageChannel.jsm

```diff
@@ -54,19 +82,35 @@ class MessageManager {
     };
   }
 
+  /**
+   * onActionFromContent - Handler for actions from a content processes
+   *
+   * @param  {object} action  A Redux action
+   * @param  {string} targetId The portID of the port that sent the message
+   */
   onActionFromContent(action, targetId) {
     this.dispatch(ac.SendToMain(action, {fromTarget: targetId}));
   }
 
+  /**
+   * broadcast - Sends an action to all ports
+   *
+   * @param  {object} action A Redux action
+   */
   broadcast(action) {
     this.channel.sendAsyncMessage(this.outgoingMessageName, action);
   }
 
+  /**
+   * send - Sends an action to a specific port
+   *
+   * @param  {obj} action A redux action; it should contain a portID in the meta.toTarget property
+   */
   send(action) {
     const targetId = action.meta && action.meta.toTarget;
     const target = this.getTargetById(targetId);
     if (!target) {
-      Cu.reportError(new Error(`Tried to send a message to a target (${targetId}) that was no longer around.`));
+      // The target is no longer around - maybe the user closed the page
```

Comment:
Sometimes a target is not around anymore for expected reasons, so this isn't an error state